src/lwc/geFormWidget/geFormWidget.js

```diff
@@ -66,29 +68,12 @@ export default class GeFormWidget extends LightningElement {
         return WIDGET_LIST.indexOf(this.element.componentName) < 0
     }
 
-    get totalAmount() {
-        return hasNestedProperty(this.widgetData, 'donationAmount') ? this.widgetData.donationAmount : 0;
-    }
-
-    /**
-     * @description This method should return a list of DataImport__c field api names
-     *              that the widget uses so that parent components (geFormSection) know
-     *              which fields to include when calling this component's load() function.
-     * @returns An Array of field names used by this component.
-     */
     @api
-    get allFieldsByAPIName() {
-        if (this.widgetComponent.allFieldsByAPIName) {
-            return this.widgetComponent.allFieldsByAPIName;
-        } else {
-            return null;
+    get paymentToken() {
```

Comment:
Should this function contain any checks to make sure the widget is of the Elevate card widget type?

Reply:
The check to make sure the widget is an elevate widget is done in `geFormSection` just before this getter is called.